docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+Querying data in Kingfisher Views
+=================================
+
+Summary tables in Kingfisher Views are generated from data in `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/>`__, which is organized by collection. Each collection contains either data downloaded from an OCDS publisher using a `Kingfisher Scrape <https://kingfisher-scrape.readthedocs.io/en/latest/>`__ spider, or loaded locally.
+
+A Kingfisher Views schema can contain summaries of one or more collections.
+
+OCDS data can be published using either `releases or records <https://standard.open-contracting.org/latest/en/getting_started/releases_and_records/>`__. Releases are point in time updates about a contracting process and a record provides an index of all the releases for a contracting process.
+
+Kingfisher Process automatically transforms OCDS releases to create compiled release collections,  containing a single compiled release per contracting process populated with the latest value of each field.
+
+Tables in Kingfisher Views can contain summaries of:
+
+* individual releases
+* compiled releases generated by Kingfisher Process
+* compiled releases published as part of an OCDS record
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled releases to begin with.
+
+List all schemas in the database
+--------------------------------
+
+Kingfisher Views creates database schemes to store summary data.
+
+The following query returns a list of schemas in the database:
+
+.. code-block:: sql
+
+  select
+    schema_name
+  from
+    information_schema.schemata;
+
+Kingfisher Views schemas are named ``view_data_*``, if you don't see any such schemas refer to :doc:`cli/index` and use the ``add-view`` command to generate a schema.
```

Comment:
Remove reference to CLI commands if this is intended for Redash users.

Reply:
I think it's still useful to have a reference to this, as I assume other types of user may also read the page.

I can put this in an admonition and add a sentence to say you need access to the server to do this if you prefer?

Reply:
If the user has access to create schemas, this should not be the first page of the docs that they read. I can't imagine a scenario where such a user would not already know by the point that they read this page that they can create schema (we should be directing such users to the home page). I think it's better to leave it out.

Reply:
Removed in https://github.com/open-contracting/kingfisher-views/pull/109/commits/e9666e4517a8ef2b8f56e53da6b6bef795c67a8b